docs/examples/test/test_examples.py

```diff
@@ -281,3 +282,31 @@ def test_websocket_rpc(self, container_factory, web_config, websocket):
 
         ws = websocket()
         assert ws.rpc('echo', value=u"hellø") == u'hellø'
+
+
+class TestConfig:
+
+    def test_config_value_not_set(self, container_factory, empty_config):
+        from examples.config_dependency_provider import (
+            Service, FeatureNotEnabled
+        )
+
+        container = container_factory(Service, empty_config)
+        container.start()
+
+        with pytest.raises(FeatureNotEnabled):
+            with entrypoint_hook(container, "foo") as foo:
+                foo()
+
+    def test_can_get_config_value(self, container_factory, empty_config):
+        from examples.config_dependency_provider import Service
+
+        config = empty_config.copy()
+        config["FOO_FEATURE_ENABLED"] = True
+
+        container = container_factory(Service, config)
+        container.start()
+
+        with entrypoint_hook(container, "foo") as foo:
+            assert "foo" == foo()
```

Comment:
it's not harmful, but it reads a bit odd and doesn't add anything, since python uses different operators for comparison and assignment, and e.g. `if a = 2:` is a syntax error